components/purchase_modal/purchase.scss

```diff
@@ -298,24 +149,156 @@
                 }
             }
 
-            .payment-note {
-                margin-top: 5px;
-                margin-bottom: 10px;
-                color: rgba(var(--sys-center-channel-color-rgb), 0.72);
+            .monthly-text {
+                align-self: center;
+                margin-left: 5px;
                 font-size: 14px;
-                font-weight: 400;
-                line-height: 20px;
+                font-weight: normal;
+            }
+
+            .plan_comparison {
+                margin-bottom: 51px;
```

Comment:
Looking at the interaction between `.plan_comparison`, `.PlanCard`, and `.planLabel`, I'd rather apply a conditional margin-top of 51px to `.PlanCard` if it receives a `planLabel` component prop. That way, we don't get an odd gap if there is no label like "Most Popular". I don't think we have any non-label cases for this right now, but since we're building it to support it, I think it makes sends to do it now and to avoid needing to change this later. Made up example below:
![no-label-scenario](https://user-images.githubusercontent.com/13738432/169844626-9807c17c-467e-442d-b9ec-20cadc193377.png)